### DISCUSSION TITLE
How to make msquic work in single thread

### DISCUSSION THREAD

**Discussion by warm-dawn** [microsoft/msquic#1680] [Q&A] [answered]
I just want it to work in a single thread or set the worker thread to 1.

**Comment by nibanks** [comment, marked as answer]
The closest thing we have right now is to use `QUIC_EXECUTION_PROFILE_TYPE_SCAVENGER` when you create your registration. This results in only a single QUIC worker thread being created to use for all QUIC connections you create.
